fixtures/showlite/sps-120.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SPS-120",
+  "shortName": "SPS-120",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Ole"],
+    "createDate": "2024-04-21",
+    "lastModifyDate": "2024-04-21"
+  },
+  "links": {
+    "manual": [
+      "https://www.kirstein.de/docs/manuals/sl-SPS-120-DMX-dt-0820.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [150, 140, 90],
+    "weight": 0.34,
```

Comment:
```suggestion
    "weight": 0.46,
```

Reply:
here it states that they are 340g, so isn't 0.34 right? 

https://www.kirstein.de/Scheinwerfer/Showlite-SPS-120-DMX-LED-Party-Spot.html

Reply:
Page 15 of the manual says 0,46 kg.  Maybe they're including the yoke in the manual (it's in the diagram), but not on the sales page?  The sales page has with/without yoke dimensions, but neither of those match the dimensions in the manual, either.  And in the manual, the "140" dimension is clearly bigger than the "150" dimension, so nothing makes any sense.

I don't trust any of these specifications, so I'm fine with putting anything here.  +/-120g isn't going to make any difference to anyone.

Reply:
You're probably right. Noone will use these in high enough numbers that +/-120g a piece will add up to anyones truss failing.

And apart from that, if you're relying on specs in a fixture you haven't written yourself (and especially a profile for a lamp this crappy) for structural calculations, you really really shouldn't be handling truss.

But another question: who is supposed to greenlight all the PRs?

Reply:
The project owner(?) will occasionally stop by and merge a bunch of reviewed fixtures.  I think I've seen 1 or 2 other people here do it, as well.  I don't have any special permissions here (nor do I really want them), so I just try to clean things up and make it easy for them.

I doubt anybody is getting paid to work on OFL for their job, so if you need something added on a deadline, I'd say your best bet is to fork the repo and use that fork.  Or if it's a simple fixture like this, there are generic mappings like `generic/rgbd-fader` that might work for you.